description: 10 seconds may not be enough, so don't kill the gpu process until 20.
author: Chad MILLER <chad.miller@canonical.com>
bug-debian: http://bugs.debian.org/781940

--- a/content/gpu/gpu_main.cc
+++ b/content/gpu/gpu_main.cc
@@ -91,7 +91,7 @@ const int kGpuTimeout = 30000;
 // infected machines.
 const int kGpuTimeout = 15000;
 #else
-const int kGpuTimeout = 10000;
+const int kGpuTimeout = 20000;
 #endif
 
 namespace content {
